packages/legacy-client/src/client/actions.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ObjectTypesFrom, OntologyDefinition } from "@osdk/api";
+import type { ObjectSet } from "./interfaces";
+import type { OsdkLegacyObjectFrom } from "./OsdkObject";
+
+export interface ValidActionParameterTypes {
```

Comment:
This is defined twice now

Reply:
Yeah this one needs to map to the legacy types, need to fix that